File: relationship-manager/recommendation-service/src/main/java/com/bank/rm/recommendation/RecommendationApplication.java
package com.bank.rm.recommendation;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.recommendation", "com.bank.rm.common"})
public class RecommendationApplication {
    public static void main(String[] args) { SpringApplication.run(RecommendationApplication.class, args); }
}

File: relationship-manager/recommendation-service/src/main/kotlin/com/bank/rm/recommendation/RecommendationApplication.kt
package com.bank.rm.recommendation

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.recommendation", "com.bank.rm.common"])
class RecommendationApplication

fun main(args: Array<String>) {
    runApplication<RecommendationApplication>(*args)
}
